features/bootstrap/FeatureContext.php

```diff
@@ -83,4 +83,16 @@ public function assertPageAddressAfterAllRedirection(string $url, int $ttl = 3,
             $this->assertPageAddressAfterAllRedirection($url, $ttl, ++$try);
         }
     }
+
+    /**
+     * @When I click the :cssElementSelector selector
+     */
+    public function clickElementSelector($cssElementSelector)
+    {
+        $field = $this->getSession()->getPage()->find('css', $cssElementSelector);
+
+        Assert::notNull($field, 'Cannot find "'.$cssElementSelector.'"');
```

Comment:
Not a big deal, but you can write easier  : 

```php
 Assert::notNull($field, "Cannot find '$cssElementSelector'");
```

Reply:
I guess the `''` are here to isolate the `$cssElementSelector` in the message :thinking: